eg/Classes/App.py

```diff
@@ -133,6 +133,7 @@ def OnExit(self):
             eg.PrintDebugNotice("The following threads did not terminate:")
             for thread in threads:
                 eg.PrintDebugNotice(" ", thread, thread.getName())
+
```

Comment:
HA sorry about that. that happened because i didn't have the thread that runs the named pipe set up as a daemon thread at first and that is where i told the thread to stop. i forgot to remove the extra line. my bad